### ISSUE TITLE
HTTPError: 404 Client Error

### ISSUE THREAD

**Issue by jiaqiizhao** [DCAN-Labs/nda-abcd-s3-downloader#60]
Hi, I encountered a HTTPError when using this downloader, it raised this error when executing this function: response = post_request(url, list(s3_path_list), auth=self.auth, error_handler=HttpErrorHandlingStrategy.reraise_status). The error message is 
requests.exceptions.HTTPError: 404 Client Error:  for url: https://nda.nih.gov/api/package/1217432/files.

**Comment by WillForan** [comment]
is this the same as #48 (need to create 2 projects one with associated data and one without)?

**Comment by jiaqiizhao** [comment]
Yes, but somehow I can't create a package with associated data, it seems to time out at updating filter cart and left me with an empty cart. I tried clearing the caches, switching to a different web browser or using a incognito window, but none of them worked and I switched to this downloader https://github.com/ABCD-STUDY/[nda-abcd-s3-downloader](https://github.com/ABCD-STUDY/nda-abcd-s3-downloader) but it has issues with creating credentials. Is there any other way to download the data?

**Comment by hough129** [comment]
Hi @jiaqiizhao - which web browser are you using?
